scripts/pre_commit_linter.py

```diff
@@ -455,7 +462,7 @@ def _check_bad_patterns(all_files):
                     print '%s --> %s' % (
                         filename, BAD_PATTERNS[pattern]['message'])
                     total_error_count += 1
-            if filename in all_js_files:
+            if filename.endswith('.js'):
```

Comment:
@seanlip - Didn't we move away from using `endsWith`due to its limited browser support ?